scalableminds/webknossos
# Improve layouting in dataset edit view

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- go to "add remote ds" and add two links
- should look like this (green lines denote the blocks that should be aligned)
![image](https://github.com/scalableminds/webknossos/assets/2486553/3a2dd549-af2c-449f-9e9f-fc451c11d911)
- on a small screen, it should switch to a single column layout:
- 
![image](https://github.com/scalableminds/webknossos/assets/2486553/1801ad1a-b0db-4212-8219-5b974884a7e9)



### Issues:
- see https://scm.slack.com/archives/C5AKLAV0B/p1686131918885679

------
(Please delete unneeded items, merge only when none are left open)
